openhtf/plugs/usb/shell_service.py

```diff
@@ -68,7 +68,6 @@
   # output.getvalue() now contains the output of the arecord command.
 """
 
-import io
```

Comment:
unused?

Reply:
It was already converted to six.StringIO.